input_manager/include/core/AbstractInputManager.hpp

```diff
@@ -38,12 +38,17 @@ class AbstractInputManager {
         }, variant_);
     }
 
+    //*For the simplicity of debug*/
+    InputManagerVariant& get_variant() {
```

Comment:
Well might be useful not only for debug, but if you want to keep incapsulation tight, remove this.

Reply:
Hid it under the `#ifdef RCBE_DEBUG`